### PR TITLE
[Navigation Tree] Move "nav up" arrow down one item

### LINKED ISSUE #3575: [User testing feedback] Tree navigation "up" arrow could be clearer
From Time Conductor and tree user testing sessions in October 2020, there was a bit of confusion on how the left "up" arrow works in the tree. Let's change the behavior a bit, in this way:

- [x] Move "up" arrow down one level, so My Items would not have a nav up arrow. @jvigliotta 
- [x] Change "up" arrow glyph to use "nav-to-parent" icon. @charlesh88

### PR DESCRIPTION
**Pull request by jvigliotta**
Moved the "nav up" arrow to show on the item below where it was previously. This will make it so it will nave up to the item ABOVE the item with the "nav up" arrow. Updated object path as well since it's built using the ancestors. Added the icon, but it will need more styling and correct css structure.

### Author Checklist
Check | Passed
-- | --
Changes address original issue? | Y
Unit tests included and/or updated with changes? | N/A
Command line build passes? | Y
Changes have been smoke-tested? | Y
Testing instructions included? | Y

closes #3575 (after @charlesh88 changes)

### CODE REVIEW

**Review by charlesh88 — CHANGES_REQUESTED** [review]
Current approach ends up hiding the nav-to-parent button that normally is visible to the left of the main view's object label:

Screen Shot 2020-12-18 at 10 24 11 AM

This needs to be fixed.

**Comment by jvigliotta** [comment]
@charlesh88 updated the css!

**Comment by charlesh88** [comment]
Reviewing again and will resolve conflicts in the branch.

**Review by charlesh88 — DISMISSED** [review]
New and improved! LGTM.

File: app.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const fs = require('fs');
 const request = require('request');
 
 // Defaults
-options.port = options.port || options.p || 8080;
+options.port = options.port || options.p || 8081;
```
Comment (nikhilmandlik) [inline app.js:18]: for testing?

**Review by nikhilmandlik — DISMISSED** [review]
Small cleanup, else LGTM

File: app.js (hunk 1 above)
Comment (deeptailor) [reply app.js:18]: thats @charlesh88 signature 😃

**Comment by deeptailor** [comment]
Looks great! Just needs one change (change port back to 8080) and we can merge it

File: app.js (hunk 1 above)
Comment (jvigliotta) [reply app.js:18]: I'll take care of this one

**Comment by jvigliotta** [comment]
> Looks great! Just needs one change (change port back to 8080) and we can merge it

Should be good now!

**Review by deeptailor — APPROVED** [review]
lgtm

**Comment by deeptailor** [comment]
## Reviewer Checklist
Changes appear to address issue? Yes
Appropriate unit tests included? N/A UI
Code style and in-line documentation are appropriate? Yes
Commit messages meet standards? Yes
Has associated issue been labelled unverified? (only applicable if this PR closes the issue) Yes

**Pull request merged**